packages/os/write-network-status.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Write network status
+# Block manual interactions with this service, since it could leave the system in an
+# unexpected state
+RefuseManualStart=true
+RefuseManualStop=true
+Before=early-boot-config.service
+After=network-online.target systemd-resolved.service
+Wants=network-online.target systemd-resolved.service
```

Comment:
This still seems like a mistake - the network should be fully configured before we reach `network-online.target` since that is its contract.

IMO:
```
[Unit]
After=systemd-networkd-wait-online.service systemd-resolved.service
Wants=systemd-networkd-wait-online.service systemd-resolved.service

...
[Install]
WantedBy=network-online.target
```

Why do we need this to run after resolved though? Can you add a comment or drop the dependency if it's not really needed?